ui/admin/app/abilities/collection.js

```diff
@@ -0,0 +1,30 @@
+import CollectionAbility from 'api/abilities/collection';
+import { inject as service } from '@ember/service';
+
+export default class OverrideCollectionAbility extends CollectionAbility {
```

Comment:
Curiosity question: this class is declared within admin and desktop with the same code. Also, it inherits from a class declared within addons. I was wondering if we could share also this class between clients? why or why not?

Reply:
That's a good point.  The reason I did this is because at the application level, capabilities depend on `ember-feature-flags` configuration which is per-app.  However the dependency itself comes from `core`, so this _might_ still work if it were pulled into `core`.